pom.xml

```diff
@@ -40,6 +40,8 @@
         <logback-classic.version>1.1.7</logback-classic.version>
         <maven-assembly-plugin>2.5</maven-assembly-plugin>
         <scalastyle-maven-plugin>0.8.0</scalastyle-maven-plugin>
+        <kafka.version>0.10.0.0</kafka.version>
```

Comment:
Having a look at the components included within the confluent platform 3.0.1 (the one we're including in PaaS), I've found out that the kafka's version is 0.10.0.1 so I guess we should bump this dep to 0.10.0.1